content/07.pathogenesis.md

```diff
@@ -388,19 +386,17 @@ Estimates of infection rates are becoming more feasible as more data becomes ava
 ### Dynamics of Transmission
 
 Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
-Thus, accurate estimates are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
-R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174].
-R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model.
-In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
+Accurate estimates of both are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
+R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x] and is one of the most important epidemiological parameters [171].
```

Comment:
```suggestion
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x] and is one of the most important epidemiological parameters [@isbn:9780691116174].
```
Is this the correct reference to add back?

Reply:
9780691116174 is correct! Thank you for catching this!